.changeset/early-camels-live.md

```diff
@@ -0,0 +1,7 @@
+---
+'@urql/exchange-graphcache': minor
```

Comment:
I'm not entirely sure if these are minor since we aren't exposing added functionality to the user apart from an added benefit when they bumped `@urql/core`

CC @kitten

Reply:
Feature addition justifies minor no? Those working on their own exchanges would likely more than a patch.

Reply:
Thinking about this, we might have to update the babel plugin to also check for `client.target` just so people who update `graphCache`/... won't crash their app if `core` isn't on par

Reply:
I think we’d want to bump `core` by a minor, but we can get away with bumping everything else by `patch` 😇

Reply:
The exchange libraries take a new argument - [that's a minor](https://semver.org/#spec-item-7)